packages/vuetify/src/components/VRating/VRating.ts

```diff
@@ -224,7 +224,7 @@ export default mixins(
 
       return this.$createElement(VIcon, this.setTextColor(this.getColor(props), {
         attrs: {
-          tabindex: -1,
+          tabindex: 0,
```

Comment:
I think you can just remove this line? VIcon renders a button when it has a click listener so it should be focusable by default.

Reply:
great! will do